dts/bindings/sensor/st,stm32-temp.yaml

```diff
@@ -28,6 +28,7 @@ properties:
         datasheet "Electrical characteristics/Operating conditions"
         STM32F1 Table 5.3.19 (min 4 mV/C, max 4.6, default 4.3)
         STM32F4 Table 6.3.21 default 2.5
+        Note: This is not used for SOC_SERIES_STM32L5X
```

Comment:
Let's be consistent and use STM32L5 instead of SOC_SERIES_STM32L5X

Reply:
EDIT: Remove this and create a new compatible dedicated to L5 temp sensor variant (and similar)